Code/Framework/AzToolsFramework/Tests/Prefab/PrefabTestFixture.cpp

```diff
@@ -76,6 +97,42 @@ namespace UnitTest
         return newEntity;
     }
 
+    AZ::EntityId PrefabTestFixture::CreateEntityWithPrefab(const char* entityName, AZ::EntityId parentId)
```

Comment:
changed to CreateEntityUnderRootPrefab, there's already another CreateEntity method and it doesn't do similar logic so makes sense to specify instead of overloading.